src/deadline/job_attachments/_utils.py

```diff
@@ -91,10 +90,13 @@ def _is_relative_to(path1: Union[Path, str], path2: Union[Path, str]) -> bool:
 
 
 def _is_windows_file_path_limit() -> bool:
-    if sys.platform == "win32":
-        ntdll = ctypes.WinDLL("ntdll")
-        ntdll.RtlAreLongPathsEnabled.restype = ctypes.c_ubyte
-        ntdll.RtlAreLongPathsEnabled.argtypes = ()
+    if sys.platform != "win32":
+        return True
+
+    import ctypes
```

Comment:
Nice fix - please fill in the PR commit template?

Reply:
You caught me in draft mode :) updated